auditbeat/docs/modules/auditd.asciidoc

```diff
@@ -206,6 +209,12 @@ that of the `audit_rules` field.
 *`backpressure_strategy`*:: Specifies the strategy that {beatname_uc} uses to
 prevent backpressure from propagating to the kernel and impacting audited
 processes.
+
+*`reload`*:: `enabled` auditbeat check auditd rule files in fixed interval 
```

Comment:
This looks like it's in the middle of the `backpressure_strategy` documentation. Think it needs to go below the `The possible values are` block.